src/pages/UserPage/UserPage.scss

```diff
@@ -0,0 +1,192 @@
+@import '../../styles/variables';
+@import '../../styles/mixins.scss';
+
+.container {
+  @include container;
+}
+
+.UserInfo {
+  padding: 60px 0;
+  background-image: url('../../assets/img/sectionLastBG.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  .container {    
+    display: flex;
+    gap: 200px;
+  }
+  .UserDescription {
+    color: #fff;
```

Comment:
Check [style guide](https://codeguide.academy/html-css.html), colors should be with 6 symbols. Also, all colors should be placed to one general file like `colors` or smth.

Reply:
fix this